### PR TITLE
{tools}[GCCcore/10.2.0] CMake v3.18.4, cURL v7.72.0, libarchive v3.4.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/CMake/CMake-3.18.4-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+]
+
+dependencies = [
+    ('ncurses', '6.2'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+    ('cURL', '7.72.0'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1d'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.18.4-GCCcore-10.2.0.eb:28]: will probably be 1.1.1h

File: easybuild/easyconfigs/c/CMake/CMake-3.18.4-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.18.4-GCCcore-10.2.0.eb:32]: osdependencies = [OS_PKG_OPENSSL_DEV]

File: easybuild/easyconfigs/c/cURL/cURL-7.72.0-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1d')
```
Comment (Micket) [inline easybuild/easyconfigs/c/cURL/cURL-7.72.0-GCCcore-10.2.0.eb:32]: 1.1.1h

File: easybuild/easyconfigs/c/cURL/cURL-7.72.0-GCCcore-10.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/cURL/cURL-7.72.0-GCCcore-10.2.0.eb:36]: osdependencies = [OS_PKG_OPENSSL_DEV]

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/eb60d8162be49a7fe0f279656b758af6 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 12 out of 13 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/664ad47b1075c4647581ca771b150781 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/85018583b59bea0b912c80a2b16b6293 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/a1e975fd6e3890545db05184084effd0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2661.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/e4f8a85e57078e4da90005a205032026 for a full test report.

**Comment by boegel** [comment]
@Micket Good to go?

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**